corehq/messaging/scheduling/templates/scheduling/conditional_alert.html

```diff
@@ -4,8 +4,8 @@
 {% load i18n %}
 
 {% block js %}{{ block.super }}
-    <script src="{% static 'data_interfaces/js/case_rule_criteria.js' %}"></script>
-    <script src="{% static 'data_interfaces/js/make_read_only.js' %}"></script>
+    {% requirejs_main 'data_interfaces/js/case_rule_criteria' %}
```

Comment:
I'd make a new module for this file rather than reusing `data_interfaces/js/case_rule_criteria`, because that module is also used by [data_interface/case_rule_criteria_base.html](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_interfaces/templates/data_interfaces/case_rule_criteria_base.html), which doesn't need the dependencies that you've added to it in this PR.

Most of the time, a page has its own unique main module, which includes a bunch of dependencies and also does some page-specific logic. In this case I don't think there will be any page-specific logic, so your `scheduling/conditional_alert` module will be basically empty, like [invoice_main.js here](https://github.com/dimagi/commcare-hq/commit/d14ba14f13d7d44e3a96940d2c72d2a1b918534d#diff-b81a32d5fee6a9c8af07b189c6a5693e).

Having the dedicated main module keeps you from introducing an unnecessary dependency between data_interfaces/case_rule_criteria_base.html and scheduling/js/create_schedule.ko - in general, be cautious of creating dependencies between django apps. They make it harder to reason about js and also make it harder for us to optimally set up the concatenated & minified bundles that are served in production.